cpp/src/hash/hashing.cu

```diff
@@ -75,12 +76,17 @@ struct row_hasher
  * @Param num_cols The number of columns in the input set
  * @Param input The list of columns whose rows will be hashed
  * @Param hash The hash function to use
+ * @Param initial_hash_values Initial hash values to combine with column hash values
```

Comment:
```suggestion
 * @Param initial_hash_values Optional array in device memory specifying an initial hash value for each column 
 * that will be combined with the hash of every element in the column. If this argument is `nullptr`, 
 * then each element will be hashed as-is.
```

Reply:
Since you're touching this function, would you mind updating the documentation to match what's in our template? https://github.com/rapidsai/cudf/blob/branch-0.6/cpp/src/examples/documentation_example.cpp

Mainly looks like just needing to add the [in] or [out] keywords for the params.